--- /usr/portage/dev-db/pgbouncer/pgbouncer-1.5.5.ebuild	2015-08-14 15:13:38.284845706 +0300
+++ pgbouncer-1.6.1.ebuild	2015-10-19 13:58:35.081813193 +0300
@@ -11,7 +11,7 @@
 
 DESCRIPTION="Lightweight connection pooler for PostgreSQL"
 HOMEPAGE="https://pgbouncer.github.io"
-SRC_URI="https://pgbouncer.github.io/downloads/${P}.tar.gz"
+SRC_URI="https://pgbouncer.github.io/downloads/files/${PV}/${P}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0"
